applications/nrf5340_audio/dfu/conf/overlay-dfu_external_flash.overlay

```diff
@@ -9,18 +9,12 @@
 };
 
 &spi4 {
-	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>, < &gpio1 10 GPIO_ACTIVE_LOW >;
+	cs-gpios = <&gpio1 10 GPIO_ACTIVE_LOW>, <&gpio0 11 GPIO_ACTIVE_LOW>, <&gpio0 17 GPIO_ACTIVE_LOW>;
```

Comment:
Discuss.

Reply:
I tested not having this here, and that will cause the chip-select pin defined in the board file to be used - which is wrong for the flash shield making the application fail. Adding only the flash shield CS will overwrite the existing pins, causing the SD reader and HW codec to not work. So we need to set all three CS pins in this file